### PR TITLE
chmod: add Portuguese translation

### PR DESCRIPTION
**Pull request by FilipaDurao**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages.pt_PT/common/chmod.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# chmod
+
+
```
Comment (sbrl) [inline pages.pt_PT/common/chmod.md:3]: Double blank line here?
```suggestion

```

**Review by sbrl — COMMENTED** [review]
Looks ok to me! Just 1 minor comment below.

Also, if you're translating multiple pages, feel free to open 1 big PR with 1 commit per page - it makes it easier to review if you're translating lots of pages :D

**Review by navarroaxel — APPROVED** [review]
LGMT after the suggestion from `sbrl` is addressed.

**Review by schneiderl — APPROVED** [review]
Looks good. Thank you for the PR. 👍

**Comment by schneiderl** [comment]
Just waiting for @sbrl requests are addressed so that we can merge.

**Review by sbrl — APPROVED** [review]
Applied fix myself due to high volume of PRs at the moment.

**Pull request merged**